samples/Directory.Build.props

```diff
@@ -1,7 +1,53 @@
-<!-- Disable any build stuff brought in by arcade for the samples.
-     By default, Arcade will bring in PackageReferences which will break any old-style projects (they will prefer those packages over packages.config) -->
 <Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+
+  <!-- When consuming the NuGet package, this is automatically added, but we must add it manually -->
+  <Import Project="$(MSBuildThisFileDirectory)../src/Microsoft.AspNetCore.SystemWebAdapters/Build/Microsoft.AspNetCore.SystemWebAdapters.props"/>
+
   <PropertyGroup>
-    <LangVersion>11</LangVersion>
+    <!-- MSBuild.SDK.SystemWeb tries to bring in default versions of the compilers, but that is now handled automatically by .NET 9 SDK -->
+    <ExcludeSDKDefaultPackages>true</ExcludeSDKDefaultPackages>
+
+    <!-- This allows the first build to be correct without any warnings on a new clone of the repo -->
+    <GeneratedBindingRedirectsAction>Overwrite</GeneratedBindingRedirectsAction>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NoWarn>
+      $(NoWarn);
```

Comment:
NIT: typically you would instead create an .editorconfig or .ruleset where you define which warnings should trigger.

Reply:
I agree, but with this it was quick and dirty to remove warnings and I'll follow up with a proper fix to the code patterns